app/services/idv/profile_maker.rb

```diff
@@ -21,13 +21,14 @@ def save_profile(
       gpo_verification_needed:,
       in_person_verification_needed:,
       selfie_check_performed:,
+      proofing_components:,
       deactivation_reason: nil
     )
       profile = Profile.new(user: user, active: false, deactivation_reason: deactivation_reason)
       profile.initiating_service_provider = initiating_service_provider
       profile.deactivate_for_in_person_verification if in_person_verification_needed
       profile.encrypt_pii(pii_attributes, user_password)
-      profile.proofing_components = current_proofing_components
+      profile.proofing_components = proofing_components.to_json
```

Comment:
`proofing_components` is a jsonb column (not a string column) so we want to give it a hash instead of a string
```suggestion
      profile.proofing_components = proofing_components.as_json
```

(for context we already cleaned this up once in https://github.com/18F/identity-idp/pull/5728)

Reply:
I see, I'll admit to not realizing `as_json` isn't an alias for `to_json`. Willfix!

Reply:
I went ahead and remove `to_json`. I didn't swap in `as_json` since `proofing_components` here is already a hash generated by `Idv::ProofingComponents`.